src/test/resources/examples/queryParameters/api.yaml

```
@@ -0,0 +1,42 @@
+openapi: "3.1.0"
+info:
+  title: "API Example"
+  version: "1.0"
+paths:
+  /query-params:
+    get:
+      summary: "GET with required query parameter"
+      parameters:
+        - $ref: "#/components/parameters/PrimitiveTypeQueryParam"
+        - $ref: "#/components/parameters/EnumQueryParam"
+      responses:
+        200:
+          description: "successful operation"
+          content:
+            application/vnd.custom.media+json:
+              schema:
+                $ref: "#/components/schemas/QueryResult"
```

Super minor thing, but avoids the `TypedApplicationCall<Any>` in the generated code.
```suggestion
                $ref: "#/components/schemas/QueryParamsResult"
```

---

Nice catch 👍